toast-tk-runtime/src/main/java/io/toast/tk/runtime/parse/FileHelper.java

```diff
@@ -0,0 +1,55 @@
+package io.toast.tk.runtime.parse;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+
+/**
+ * Helper class for getting files content.
+ * <p>
+ * Created by Nicolas Sauvage on 02/08/2016.
+ */
+public class FileHelper {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1118)